source/boards/get-started-with-boards.rst

```diff
@@ -0,0 +1,66 @@
+
+Create a board
+--------------
+
+When you first start using Boards, you need to create a board. You can select a template from the options provided or you can create an empty board. The only difference is that a template will be populated with sample cards and properties and in an empty template you start from scratch.
+
+To find a board, open Boards and select the drop-down menu in the sidebar. Each board is linked to the channel it was created in. This includes direct messages and group messages.
+
+To learn more about boards and templates, visit XXX.
+
+Create a card
+-------------
+
+Cards are like sticky notes. You can arrange them in various ways, using different views. You can also give them mutiple areas of information called properties. Adding and using properties makes it easier to organize and find cards, using filters based on card properties.
+
+Once you've chosen a board template, you'll find pre-populated cards. You can use and edit those, or create new ones. Use the Description field to add details, and you can use Markdown to add formatting. You can add comments to a card in the Comments section.
+
+Select an existing card to open it or create a new card.
+
+Description
+-----------
+
+Use the description to provide context around the reason for the card. Cards are visible to anyone who can access the board, so if there's sensitive information it's best not to include it here. You can add links to external documentation, as well as at-mention teammates.
+
+Comments
+--------
+
+Comments are a helpful way to add additional information to a card, and comments are time and date stamped so they're easy to track. Comments are listed in descending order from most recent to oldest.
+
+To add a comment, select the **Comment** field. When you're finished, select **Save**.
+
+Properties
+----------
+
+You can create properties for your cards that suit the way you're using them. If you're planning a project, you might want to have properties such as **Due date**, **Assigned to**, and **Priority**. Properties are displayed on the card when you open it, and you can also choose to display your cards based on their properties, using filters.
+
+Properties can be renamed and deleted.
+
+To learn more about properties, visit XXX.
+
+Views
+-----
+
+View are a great way to assess your board from various angles. Some views are linear, while others are closer to Kanban style. Views with a more linear display, such as the task view, display less context but are easier to scroll through. Use a view such as **By Status** to see all the cards, presented in swimlanes, with more details, in the form of properties, displayed.
+
+You can also choose views such as a calendar view, which gives you a view of your work overlaid on a calendar grid.
+
+To learn more about views, visit XXX.
+
+Filters and sorting
```

Comment:
- What's the difference between filters and sorting? 
- How are filters defined?
- When would you filter and when would you sort?